### PR TITLE
Feature/io pay second page

### PR DESCRIPTION
**Pull request by Infectedmachine**
#### List of Changes

Setup checkout payment summary page
Setup redux
Edit all inline styles to adapt to mui/italia theme

#### How Has This Been Tested?

Manual test in browser

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: io-pay-portal-fe/src/components/FormButtons/FormButtons.tsx [hunk 1]
```diff
@@ -16,31 +16,27 @@ export function FormButtons(props: {
...
         container
         spacing={2}
       >
         <Grid
-          xs={3}
-          md={3}
-          lg={3}
-          xl={3}
+          xs={4}
+          md={4}
```
Comment (vito80ba) [inline io-pay-portal-fe/src/components/FormButtons/FormButtons.tsx]: Are you sure that it's necessary set also md-lg-xl ? MUI is mobile first, you could set only the "base value" if it's the same for all viewports

File: io-pay-portal-fe/src/components/FormButtons/FormButtons.tsx [hunk 2]
```diff
@@ -52,17 +48,16 @@ export function FormButtons(props: {
...
           >
             {t(props.cancelTitle)}
           </Button>
         </Grid>
         <Grid
-          xs={6}
-          md={6}
-          lg={6}
-          xl={6}
+          xs={8}
```
Comment (vito80ba) [inline io-pay-portal-fe/src/components/FormButtons/FormButtons.tsx]: Same as above

File: io-pay-portal-fe/src/components/FormButtons/FormButtons.tsx [hunk 3]
```diff
@@ -52,17 +48,16 @@ export function FormButtons(props: {
...
+          md={8}
+          lg={8}
+          xl={8}
           style={useSmallDevice() ? { paddingTop: 0 } : {}}
```
Comment (vito80ba) [inline io-pay-portal-fe/src/components/FormButtons/FormButtons.tsx]: Could you write this in the pattern expected avoing style? Like
`pt={xs: 1, sm: 0}`

File: io-pay-portal-fe/src/routes/PaymentPage.tsx [hunk 4]
```diff
@@ -1,30 +1,33 @@
...
   const { t } = useTranslation();
   const [modalOpen, setModalOpen] = React.useState(false);
 
   return (
-    <main style={{ padding: "1rem 0" }}>
-      <h1 style={{ marginBottom: 0 }}>{t("checkoutForm.title")}</h1>
-      <p style={{ marginTop: 8, marginBottom: 8 }}>
-        {t("checkoutForm.description")}
-      </p>
+    <main style={{ padding: "3rem 0" }}>
```
Comment (vito80ba) [inline io-pay-portal-fe/src/routes/PaymentPage.tsx:14]: Try to avoid `style`

**Review by vito80ba — CHANGES_REQUESTED** [review]
Great work, but there's stille something.
But now the code is more readable

File: io-pay-portal-fe/src/components/FormButtons/FormButtons.tsx (hunk 3 above)
Comment (Infectedmachine) [reply io-pay-portal-fe/src/components/FormButtons/FormButtons.tsx]: on xs screen pt={xs: 1, sm: 0} doesn't overwrite the mui grid item default padding of 16px, while it does with style. 
will try a better solution.

**Review by vito80ba — APPROVED** [review]

**Review by vito80ba — APPROVED** [review]

**Pull request merged**